src/NexusMods.DataModel/GameRegistry/Registry.cs

```diff
@@ -27,6 +25,11 @@ public class Registry : IGameRegistry, IHostedService
     private readonly ReadOnlyObservableCollection<GameInstallation> _installedGames;
     private readonly ILogger<Registry> _logger;
     private readonly IEnumerable<ILocatableGame> _games;
+    private bool _isInitialized;
+    
+#if DEBUG
+    private int _initThreadId;
+#endif
```

Comment:
Remove #if DEBUG . Either leave all the code and do a Debug assert at the end or remove all the debug code.

Reply:
>  waste 8 bytes of RAM
Insert shocked expression here.

I think we can spare 8 bytes in release.

Reply:
😭 

Think of all of the collective seconds that would be saved from all the hundreds of thousands of people people shaving nanoseconds from App startup on the daily.

*sobbbbb*

Yeah, don't worry, I'll get it sorted when I get back to my machine later.